x-pack/test/functional/apps/endpoint/alert_list.ts

```diff
@@ -21,6 +21,8 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
 
     it('loads the Alert List Page', async () => {
       await testSubjects.existOrFail('alertListPage');
+      const alertsTitle = await testSubjects.getVisibleText('alertsViewTitle');
+      expect(alertsTitle).to.equal('Alerts');
```

Comment:
It might be better to move it to a separate `it` block:
```javascript
it('contains a page title', () => {});
```

Reply:
I can update this, but I disagree on the premise.  

I have a separate discussion I'm going to post to our team and possibly the larger group about our test groupings.  We're loading the page 3-5 times and thus far doing nothing interactive and there isn't even any data - it can be streamlined, and the tests run faster, if we choose to.    

For now I'd propose we leave this as is, the word 'Alerts' is an integral part of the initial page load as the test case assesses.   @peluja1012

Reply:
It would seem that this `expect` should be in it's own `it` block based on how this `describe` block is already structured.  Perhaps
`it('includes Alerts title', ...`

Reply:
It feels immaterial to me as I hope every page that loads has a title to be assessed. I can update it all the same, tho.  Most importantly I am interested in driving the conversation further on test code structure best practices.

Reply:
ok, i updated the test to a new 'it' block.  i still disagree its needed or best, but I got 2 opinions on this that concurred, so its fine by me as its still fairly immaterial.  Testrail updated to include a new test.  Please re-review, @charlie-pichette @peluja1012

Reply:
My reasoning for having it in a separate `it()` block was that it would be easier to determine on a failed test execution that the test failed because the `Alerts` title changed as opposed to the page not rendering at all. But I can also see the point that the title could be considered an integral part of the page loading.